src/app/workflow/info-tab/info-tab.service.ts

```diff
@@ -85,7 +85,7 @@ export class InfoTabService {
    * Additionally, the webservice does not appear to understand starredUsers which causes an error.
```

Comment:
Appears that this is still true, noticed that I can't update topic settings for an entry with starred users so I had to use this function